core/js/src/object.ts

```diff
@@ -21,6 +25,7 @@ export type OtherExperimentLogFields = {
   metadata: Record<string, unknown>;
   metrics: Record<string, unknown>;
   datasetRecordId: string;
+  origin?: z.infer<typeof objectReferenceSchema>;
```

Comment:
Pretty sure it doesn't need to be marked optional, similar to all the other fields

Reply:
good catch